package.json

```diff
@@ -171,6 +174,7 @@
         "./website:lint",
```

Comment:
GLOBAL: We should add all the `"./packages/spectacle-mdx-loader:*"` here on all the applicable wireit tasks.

Reply:
I've updated these, let me know if I've missed anything!